README.md

```diff
@@ -1,2 +1,96 @@
 # d2-logger
-DHIS2 library that allows a certain application to register events in a DHIS2 program
+
+DHIS2 library that allows a certain application to register logs as events in a DHIS2 program or display logs on the console.
+
+## Usage
+
+There are two types of logger output:
+
+1. Using a DHIS2 events program to register the logs:
+
+    You will need to have and existing event program in DHIS2 with the following data elements with value type Text: Message and MessageType. MessageType would be and option set with the following options: "Error", "Warn", "Success", "Info" and "Debug".
+
+    Therefore, the following configuration will be passed to the logger:
+
+    ```typescript
+    import { Logger } from "@eyeseetea/d2-logger";
+
+    const logger = new Logger();
+
+    await logger.init({
+        type: "program",
+        debug: debug,
+        baseUrl: baseUrl,
+        auth: auth,
+        organisationUnitId: organisationUnitId,
+        programId: programId,
+        loggerDataElements: {
+            messageId: messageId,
+            messageTypeId: messageTypeId,
+        },
+    });
+
+    logger.debug("This is a Debug message");
+    logger.info("This is an Info message");
+    logger.success("This is a Success message");
+    logger.warn("This is a Warn message");
+    logger.error("This is an Error message");
+    ```
+
+    Notice:
+
+    - Please note that `auth` is not mandatory if it's used in the DHIS2 app instead of in a script.
+    - Please note that if the `organisationUnitId` is not added to the configuration, then the logs will be registered in the DHIS2 program in the global organization unit.
+    - If `debug` is `true`, then in addition to registering the logs in the DHIS2 program, they will also be displayed on the console.
+
+2. Displaying the logs only in the console:
+
+    ```typescript
+    import { Logger } from "@eyeseetea/d2-logger";
+
+    const logger = new Logger();
+
+    logger.init({
+        type: "console",
+    });
+
+    logger.debug("This is a Debug message");
+    logger.info("This is an Info message");
+    logger.success("This is a Success message");
+    logger.warn("This is a Warn message");
+    logger.error("This is an Error message");
+    ```
+
+## Development
+
+```bash
+$ yarn install
+$ yarn build
+$ cd build
+$ yarn link
+```
+
+On another app:
+
+```bash
+$ yarn link "@eyeseetea/d2-logger"
+```
+
+## Tests
+
+```bash
+$ yarn test
+```
+
+## Code quality
+
+```bash
+$ yarn code-quality
+```
+
+## Publish
+
+```bash
+$ yarn prerelease
+$ yarn release [--tag beta] [--patch | --minor | --major]
```

Comment:
yarn release calls scripts/publish, which has no options. The update of version (or the use of beta), are manual, changing package.json -> version. Example of beta: `"version": "0.1.0-beta.1",`

Reply:
I've deleted options and added this instruction to the readme